django/db/models/deletion.py

```diff
@@ -279,6 +329,8 @@ def delete(self):
                         sender=model, instance=obj, using=self.using
                     )
 
```

Comment:
chop blank line here and in the similar code below (the parenthesis on its own line provides whitespace)